fairlearn/preprocessing/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Microsoft Corporation and contributors.
+# Licensed under the MIT License.
+
+"""This module contains algorithms implementing preprocessing steps that might help
+filter information that correlates with sensitive attributes.
```

Comment:
I guess for right now it's fine, but in general preprocessing can do things other than filtering information, right? I remember Sanmi Koyejo's NeurIPS 2019 presentation which summarized ideas in the area of representation learning for fairness, for example, which could also fit in here. http://sanmi.cs.illinois.edu/documents/Representation_Learning_Fairness_NeurIPS19_Tutorial.pdf

Reply:
Will have a look at that video. I'll keep the information as is for now, it feels best to update it once other tools go in there. There are many preprocessing steps possible but since I can't predict all of them now, it's probably better to iteratively add them.